### PR TITLE
Implements namespaced model for the dashboard and routes generators

### PR DESCRIPTION
**Pull request by bikramwp**
Reference: https://github.com/thoughtbot/administrate/pull/871
* support namespaced model for the dashboard generator
* support namespaced model for the routes generator

### CODE REVIEW

File: lib/generators/administrate/routes/routes_generator.rb [hunk 1]
```diff
@@ -63,6 +58,26 @@ def namespaced_models
         database_models.select { |model| model.to_s.include?("::") }
       end
 
+      def generate_resource_routes(resource)
+        if resource.include?("/")
+          parts = resource.split("/")
+          generate_nested_resource_routes(parts, 0)
```
Comment (nickcharlton) [inline lib/generators/administrate/routes/routes_generator.rb]: What do you think about the number here (and below, in the implementation) being constants? This should help us avoid having some magic numbers spread about this file.

**Comment by ACPK** [comment]
@nickcharlton  - Without the support for namespaced models, Administrative wouldn't be the best choice for me. Hence, thank you for your PR! 

Do you know what the status of this PR is as it's been a few months?

**Comment by sedubois** [comment]
@nickcharlton is there any chance of this important PR being merged, or has development of new Administrate features been stopped?

I have a project using Globalize and PaperTrail, which make use of namespaced models (e.g `Thought::Translation` or `PaperTrail::Version`), so this is a blocking issue for us.

Thanks!

**Comment by sedubois** [comment]
OK I managed to workaround the generator issues, so looks like might be able to use this gem!
https://github.com/thoughtbot/administrate/issues/1291#issuecomment-545054789

**Comment by pablobm** [comment]
Closed due to lack of activity.

**Pull request closed without merging**